src/piksel/baseapp.cpp

```diff
@@ -70,6 +70,13 @@ void BaseApp::start() {
         exit(EXIT_FAILURE);
 #endif
     }
+#ifndef __EMSCRIPTEN__
+    if(fullscreen && width == -1){
```

Comment:
nit: missing space after if and before {
if i understand this correctly, you want the user to have the option to either provide custom width/height for fullscreen mode or determine the default fullscreen resolution by checking for width == -1. i would move the check around the assignments of width and height respectively.

```suggestion
    if (fullscreen) {
      //...
      if (width == -1) {
        width = mode->width;
      }
      if (height == -1) {
        height = mode->height;
      }
```